pkg/registry/fixedRegistry.go

```diff
@@ -0,0 +1,41 @@
+package registry
+
+import "sync"
+
+// TODO: Delete this or move to a test file
+type FixedNodeRegistry struct {
+	nodes                     []Node
+	newNodeNotifier           *notifier[[]Node]
+	changedNodeNotifiers      map[uint16]*notifier[Node]
+	changedNodeNotifiersMutex sync.Mutex
```

Comment:
I don't think these notifiers ever get triggered?

Reply:
This whole thing is getting deleted pretty soon